Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/AssetFolderThumbnailView.cpp

```diff
@@ -162,7 +163,8 @@ namespace AzQtComponents
             QIcon icon;
             if (const auto& path = qVariant.value<QString>(); !path.isEmpty())
             {
-                icon.addFile(path, rect.size(), QIcon::Normal, QIcon::Off);
+                icon.addFile(path, imageRect.size(), QIcon::Normal, QIcon::Off);
+                AZ_Assert(!icon.isNull(), "Asset Browser Icon not found for file '%s'", path.constData());
```

Comment:
The [QString::constData](https://doc.qt.io/qt-6/qstring.html#constData) function returns a QChar*, not a const char*.
A QChar is a 16 bit short and this assert will never output correctly.
```suggestion
                AZ_Assert(!icon.isNull(), "Asset Browser Icon not found for file '%s'", path.toUtf8().constData());
```